pywbem/_listener.py

```diff
@@ -632,6 +632,11 @@ def __init__(self, host, http_port=DEFAULT_LISTENER_PORT_HTTP,
 
             `None` means not to use a private key file. Setting up a port
             for HTTPS requires specifying a private key file.
+
+          listener_id (:term:`string`):
+            If not `None` a string that prefixes the name property of filters
+            and handler instances to help the listener identify its filters and
+            handler instances in a WBEMServer.
```

Comment:
On term "WBEMServer": Here we talk about the actual WBEM server, not about the class `WBEMServer`, so I suggest to change this to "WBEM server.".